config/vufind/config.ini

```diff
@@ -1281,6 +1281,13 @@ rfr_id          = vufind.svn.sourceforge.net
 ; "demo" generates fake values for use in testing the embed setting below).
 ;resolver        = sfx
 
+; If the Alma OpenURL resolver returns reasons to filter specific resource links,
+; these reasons may be ignored by adding the "filter reason" to this list.
+; List of filter reasons that are ignored (displayed regardless of filtering).
+; You may also use a comma-separated list of filter reasons.
+; (default is "" if commented out: non filter reason ignored)
+ignoredFilterReasons            = "Date Filter"
```

Comment:
Would it make sense to use an array-type setting here, rather than a delimited list? e.g.

```
ignoredFilterReasons[] = "reason 1"
ignoredFilterReasons[] = "reason 2"
```

rather than

```
ignoredFilterReasons = "reason 1,reason2"
```

?

This would require less parsing to deal with in the driver, and might avoid potentially problematic edge cases. You could support having the setting set to false (`ignoredFilterReasons = false`) to disable filtering.

Also, is there a strong reason to change the default to "no filters" rather than allowing the driver to have default settings? Making this change could introduce unexpected changes in behavior if people fail to update their configurations when upgrading. I'm not sure how likely that is to be a problem but wanted to raise the question.

Reply:
Also, a suggested rewording of the comment that might be more "future-proof":

```
; Some link resolver drivers can filter resource links based on specific criteria.
; This setting indicates which filter reasons should NOT be used to exclude
; resource links.
; [note about format, depending on discussion above]
; [note about defaults, depending on discussion above]
; This is currently ONLY supported by the Alma resolver driver.
; Legal values include: Date Filter (any others of note?)
```

Reply:
There is another Filter Reason I found "Available for - the ip groups are not part of the service AF groups" but unfortunately I wasn't able to find any documentation about those reasons.